.github/workflows/owasp.yaml

```diff
@@ -16,33 +16,15 @@ jobs:
         with:
```

Comment:
You'll need to merge the latest version of dev into your branch - this file shouldn't be in your changeset.